README.md

```diff
@@ -1,2 +1,31 @@
 # hermes-host
 The Hermes host. Contains logic for peer to peer streaming of applications and games. Run on individual 'hosts'. Matchmaking between hosts and clients is done by 'hermes-core'
+
+
+# Includes
```

Comment:
File structure suggestion:
```
- hermes-host
--- src
------ client.js
------ host.js
--- client.html
--- host.html
```
In the future we would remove 'client' as that would be part of hermes-ui

Reply:
Nice remark!

Reply:
done :D